src/lib/getFigmaLinks.ts

```diff
@@ -0,0 +1,10 @@
+import { mapValues } from 'lodash'
+
+export function getFigmaLinks(nodeIds: Record<string, string>) {
+  /*
+    https://www.figma.com/developers/api#get-file-nodes-endpoint
+    The node Id and file key can be parsed from any Figma node url: https://www.figma.com/file/:key/:title?node-id=:id.
+    */
+  const figmaLinkBase = 'https://www.figma.com/file/JrmVrSdKCHUZBQFEB3xsjJ/Figma-Design-Library?node-id='
+  return mapValues(nodeIds, (nodeId) => `${figmaLinkBase}${nodeId}`)
```

Comment:
Would be good to use a library (or similar) to build the query params e.g. the built-in URLSearchParams:

```
> const searchParams = new URLSearchParams({"node-id": "123"});
> searchParams.toString()
'node-id=123'
```

Or URL:

```
> const url = new URL('https://www.figma.com/file/JrmVrSdKCHUZBQFEB3xsjJ/Figma-Design-Library')
> url.searchParams.set('node-id', '123')
> url.href
'https://www.figma.com/file/JrmVrSdKCHUZBQFEB3xsjJ/Figma-Design-Library?node-id=123'
```

(Just to avoid any future problems with escaping etc.)